glide.yaml

```diff
@@ -0,0 +1,19 @@
+package: github.com/giantswarm/azure-operator
```

Comment:
I think the deps are correct but let me know if you think external deps need pinning.